src/fosslight_util/write_excel.py

```diff
@@ -82,18 +82,29 @@ def remove_empty_sheet(sheet_items):
     return success, final_sheet_to_print
 
 
-def get_header_row(sheet_name, sheet_content):
+def get_header_row(sheet_name, sheet_content, extended_header):
```

Comment:
Let's change` extended_header `to `extended_header={}`.

Reply:
Changed as suggested.